### PR TITLE
[preprocess]remove max_fee and mulltiplier in preprocess request

### PR DESCRIPTION
**Pull request by jingweicb**
Fixes # .

### Motivation
these two fields are conflict add complex in our internal services
and sometimes make the return dynamic fee unreliable

### CODE REVIEW

File: api.json [hunk 1]
```diff
@@ -1996,7 +1996,7 @@
...
+       "description":"ConstructionPreprocessRequest is passed to the `/construction/preprocess` endpoint so that a Rosetta implementation can determine which metadata it needs to request for construction. Metadata provided in this object should NEVER be a product of live data (i.e. the caller must follow some network-specific data fetching strategy outside of the Construction API to populate required Metadata). If live data is required for construction, it MUST be fetched in the call to `/construction/metadata`. In the case that the caller provides both a max fee and a suggested fee multiplier, the max fee will set an upper bound on the suggested fee (regardless of the multiplier provided).",
```
Comment (GeekArthur) [inline api.json:1999]: > In the case that the caller provides both a max fee and a suggested fee multiplier, the max fee will set an upper bound on the suggested fee (regardless of the multiplier provided)."

Do we need this description as we already remove max fee and suggested fee multiplier?

Comment (jingweicb) [reply api.json:1999]: yes, the discription is for other fields in preprocess

Comment (GeekArthur) [reply api.json:1999]: I think the description I _commented_ above only include max fee and suggested fee multiplier, why we need to keep it for other fields?

Comment (jingweicb) [reply api.json]: updated, pls have a look

**Review by GeekArthur — APPROVED** [review]

**Pull request merged**